lib/backend/k8s/resources/globalnetworkpolicies.go

```diff
@@ -30,61 +30,61 @@ import (
 )
 
 const (
-	SystemNetworkPolicyResourceName = "systemnetworkpolicies"
-	SystemNetworkPolicyTPRName      = "system-network-policy.alpha.projectcalico.org"
-	SystemNetworkPolicyNamePrefix   = "snp.projectcalico.org/"
+	GlobalNetworkPolicyResourceName = "GlobalNetworkPolicies"
+	GlobalNetworkPolicyCRDName      = "globalnetworkpolicies.crd.projectcalico.org"
+	GlobalNetworkPolicyNamePrefix   = "crd.projectcalico.org/"
```

Comment:
I think what we want to do is remove the `GlobalNetworkPolicyNamePrefix` altogether.

Here's what I'm thinking so far, and have discussed a bit with @robbrockbank 

A NetworkPolicy with name "foo" should show up like this in `calicoctl get policies`:
- `knp.default.foo`

A GlobalNetworkPolicy with name "foo" should show up like this in `calicoctl get policies`:
- `foo`